### PR TITLE
TM-3457 - AIM Button Alignment

### PR DESCRIPTION
**Pull request by pveland621**
Before:
image

After (Flexbox edition):
image

ACs:
1. Save button is correctly aligned with end of legs box

### CODE REVIEW

File: src/sass/_agendaItemMaintenance.scss [hunk 1]
```diff
@@ -76,13 +76,15 @@
   .ai-maintenance-header {
     background-color: $tm-white-smoke-light;
     padding-bottom: 20px;
+    display: flex;
+    flex-wrap: wrap;
```
Comment (SidelineCory24) [inline src/sass/_agendaItemMaintenance.scss:80]: can remove this

File: src/sass/_agendaItemMaintenance.scss [hunk 2]
```diff
@@ -94,7 +96,10 @@
     .back-save-btns-container {
       padding-top: 2.5rem;
       min-height: 70px;
-      width: 95%;
+      justify-content: space-between;
+      padding-bottom: 10px;
+      display: flex;
+      width: 100%;
```
Comment (SidelineCory24) [inline src/sass/_agendaItemMaintenance.scss:102]: can remove everything but the `width: 100%`

File: src/sass/_agendaItemMaintenance.scss [hunk 3]
```diff
@@ -106,7 +111,6 @@
         color: $primary-blue;
         background-color: $color-white;
         padding: 10px 20px;
-        float: right;
```
Comment (SidelineCory24) [inline src/sass/_agendaItemMaintenance.scss:109]: add this back in

Comment (pveland621) [reply src/sass/_agendaItemMaintenance.scss]: Done

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go!🚀

File: src/sass/_agendaItemMaintenance.scss [hunk 4]
```diff
@@ -82,7 +82,7 @@
     }
 
     .half-width {
```
Comment (elizabeth-jimenez) [inline src/sass/_agendaItemMaintenance.scss:84]: may want to update this class name now that it is full width

Comment (SidelineCory24) [reply src/sass/_agendaItemMaintenance.scss:84]: 🦅

Comment (SidelineCory24) [reply src/sass/_agendaItemMaintenance.scss:84]: can be removed altogether with `.back-save-btns-container` change

**Comment by SidelineCory24** [comment]
simplifying the styling allowed for the removal of unused styling, thus turning into removing some unused logic to trigger the styling

**Pull request merged**